wdio.conf.js

```diff
@@ -0,0 +1,182 @@
+// Stop node from complaining about fake memory leaks at higher concurrency
+require("events").defaultMaxListeners = 20;
```

Comment:
Should be single quotes through this whole file

Reply:
Oops, I ran `npx eslint --fix` but only did it to the test file.